commcare/src/main/java/org/motechproject/commcare/pull/CommcareFormImporterImpl.java

```diff
@@ -111,11 +112,13 @@ public void run() {
                         }
                     } catch (RuntimeException e) {
                         LOGGER.error("Error while importing forms", e);
+                        LOGGER.error("{} of {} forms imported.", importCount, totalCount);
                         handleImportError(e, configName);
                     }
                 } while (importInProgress && hasMore);
 
-                LOGGER.info("Form import finished");
+                LOGGER.info("Form import finished. {} of {} forms imported. Last form xmlns to be imported was {}", importCount, totalCount, lastFormXMLNSToBeImported);
```

Comment:
The Last form XMLNS to be imported was meant to be logged when there's an error, right?